src/Contexts/notesContext/notes-context.js

```diff
@@ -25,19 +29,24 @@ const NotesProvider = ({ children }) => {
           const responce = await axios.get("/api/notes", {
             headers: { authorization: token },
           });
-          console.log("from await");
           noteDispatch({ type: SET_NOTES, payload: responce.data.notes });
         } catch (error) {
           console.log(error);
         }
       })();
     } else {
-      console.log("from else");
       noteDispatch({ type: SET_NOTES, payload: [] });
     }
   }, [isLoggedIn]);
+
+  const notes = removeTrashedNotes(noteState.notes);
+  const bookMarkedNotes = getBookMarkedNotes(notes);
+  const mainNotes = getNoNBookMarkedNotes(notes);
+
```

Comment:
You can change the name of `getNoNBookMarkedNotes` to `mainNotes` in utils as well.